### PR TITLE
chore/bump oa 0.3.2

### PR DESCRIPTION
**Pull request by 0xArdi**
This PR bumps the El Collectooorr to `open-autonomy@v0.3.2`.

### CODE REVIEW

File: .github/workflows/main_workflow.yml [hunk 1]
```diff
@@ -72,7 +72,7 @@ jobs:
     strategy:
       matrix:
         os: [ ubuntu-latest ]
-        python-version: [ "3.10" ]
+        python-version: [ "3.10.6" ]
```
Comment (0xArdi) [inline .github/workflows/main_workflow.yml:75]: Pinning python version to avoid static checks failing.

Comment (0xArdi) [reply .github/workflows/main_workflow.yml:75]: I notice this is not done on open-autonomy.

File: Pipfile [hunk 2]
```diff
@@ -9,54 +9,20 @@ name = "pypi"
 
 [dev-packages]
 # we fix exact versions as it's sufficient to have at least one set of compatible dependencies for development
-aiohttp = "==3.7.4.post0"
```
Comment (DavidMinarsch) [inline Pipfile:12]: See other PR. -a bunch of these would be needed in packages.

File: tox.ini [hunk 3]
```diff
@@ -24,12 +24,12 @@ deps =
     hypothesis==6.21.6
```
Comment (DavidMinarsch) [inline tox.ini:24]: See comments on other PR re matching Pipfile

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**